src/data/defaults.txt

```diff
@@ -835,6 +835,9 @@ user	lastGuildStoreOpen	-1
 user	lastGuyMadeOfBeesReset	-1
 user	lastFratboyCall	-1
 user	lastFriarCeremonyAscension	-1
+user	lastFriarElbowNC	0
```

Comment:
Why not -1?

Reply:
I feel like that's a less informative number than 0; under normal circumstances, if `Location.get("the dark elbow of the woods").turnsSpent - get("lastFriarEblowNc") === 5`, you're due for a pity NC. Setting the default to -1 doesn't accomplish that.

Reply:
The current default informs me that the last friar elbow NC was on turn zero, which is incorrect...

Reply:
Your code snippet can just be

```ts
Location.get("the dark elbow of the woods").turnsSpent - Math.max(get("lastFriarEblowNc"), 0) === 5
```

I realise this is more awkward to code but I'd rather not make confusing prefs